arkouda/numpy/_numeric.py

```diff
@@ -103,6 +103,21 @@ class ErrorMode(Enum):
     return_validity = "return_validity"
 
 
+# merge_where comes in handy in functions that include a "where" argument.
+
+def merge_where(new_pda, where, ret):
```

Comment:
I wasn't meaning to make it user-accessible, so I'll add the underscore.  By "hints," do you mean it should say what's acceptable?

Reply:
`merge_where(new_pda:<data_type>, where:<data_type>, ret:<data_type>)-> <data_type>:`

These are used for type checking and IMO we could be more consistent about using them.

Reply:
I agree.  We could stand to have an issue all about consistent error and condition handling.  I don't know that we have a problem, but I think it would still be helpful (for me, at least) to clarify exactly how we do it.